ompi/mpi/c/win_get_info.c

```diff
@@ -27,6 +27,10 @@
 
 static const char FUNC_NAME[] = "MPI_Win_get_info";
 
+static void _win_info_set (ompi_info_t *info, const char *key, int set)
```

Comment:
See C99 § 7.1.3 bullets 1-2. The first designates _[A-Z]* and __*. The second explicitly allows _[a-z]* for file-scoped identifiers.